src/llama-context.cpp

```diff
@@ -285,6 +285,8 @@ llama_context::llama_context(
         const uint32_t n_seqs = cparams.kv_unified ? 1 : cparams.n_seq_max;
         const uint32_t n_tokens = std::min(cparams.n_ctx, cparams.n_ubatch);
 
+        n_outputs = 1;
```

Comment:
Add a comment here explaining the reason. Something like:

```suggestion
        // avoid reserving graphs with zero outputs
        n_outputs = 1;
```

Maybe consider adding an assert in `graph_reserve` too.